### ISSUE TITLE
Show name of contact next to alias

### ISSUE THREAD

**Issue by zetok** [qTox/qTox#1195]
Currently name is not displayed when there's an alias set for contact. It should be changed from `Name / Alias`
to:
if `only name`:

Name

if `name & alias`:

Alias _(Name)_

**Comment by ProMcTagonist** [comment]
+1, would help alleviate the problem behind #1189

**Comment by zetok** [comment]
Also, I think that tab-autocompletion in groupchats should complete name, even if one started typing an alias.. I.e. currently it's very annoying that it autocompletes an alias of a contact, without any means of just saying their name, unless an alias is removed.

**Comment by o-jasper** [comment]
Yes please, some people just changed names, and i am not 100% who is who.

**Comment by o-jasper** [comment]
See also https://github.com/Tox/Tox-STS/issues/1, i think priority should be bumped because it allows privacy being broken and contacts being confused by mixing up the names..

Infact, imma go and try make a branch that does that, developing from master okey?
